shell/gpu/gpu_surface_metal.mm

```diff
@@ -159,16 +199,4 @@
   return std::make_unique<GLContextDefaultResult>(true);
 }
 
-void GPUSurfaceMetal::ReleaseUnusedDrawableIfNecessary() {
-  // If the previous surface frame was not submitted before  a new one is acquired, the old drawable
-  // needs to be released. An RAII wrapper may not be used because this needs to interoperate with
-  // Skia APIs.
```

Comment:
How is this dealt with in the new Diff scheme? If we never rendered it, then will its "damage area" be dealt with? I'm guessing it will because we never cleared its damage rect from `damage_`?

Reply:
The drawable is no longer stored in `next_drawable_` so there is nothing to release. In the PR the drawable is obtained before rasterization (in order to be to know accumulated damage) and stored in scoped_nsprotocol, so it will be unreffed when it gets out of scope. Before PR the drawable was obtained in a Skia callback and stored as ivar, that's why it needed to be released.